docs/version0.18.md

```diff
@@ -30,6 +30,7 @@ The following new features and notable changes since v 0.17.0 are included in th
 - [jextract now available on macOS for OpenJDK version 8](#jextract-now-available-on-macos-for-openjdk-version-8)
 - [New shared classes cache suboption to turn off timestamp checking](#new-shared-classes-cache-suboption-to-turn-off-timestamp-checking)
 - [`-Xmso` 1 MB minimum value on z/OS 64-bit](#-xmso-1-mb-minimum-value-on-zos-64-bit)
+- [-XX:+TransparentHugePage is enabled by default on Linux&reg POWER&reg and IBM Z&reg systems](#-XXTransparentHugePage-is-enabled-by-default-on-Linux-POWER-and-IBM-Z-systems)
```

Comment:
Linux is TM, use `&trade;`
Registered trademarks missing the closing ";" i.e.`&reg;`

Reply:
Fixed.